plottery.py

```diff
@@ -171,6 +171,8 @@ def __init__(self, options={}, kind=None):
             "extra_text_ypos": { "type": "Float", "desc": "NDC y position (0 to 1) for extra text", "default": 0.87, "kinds": [ "1dratio","graph"], },
 
             "extra_lines": { "type": "List", "desc": "list of 4-tuples (x1,y1,x2,y2) for lines", "default": [], "kinds": [ "1dratio","graph"], },
+            "extra_dashed_lines" : { "type": "List", "desc": "list of 4-tuples (x1,y1,x2,y2) for dashed lines", "default": [], "kinds": [ "1dratio","graph"], },
```

Comment:
you could also just extend `extra_lines` to accept 5-tuples `(x1,y1,x2,y2,linestyle)`, where linestyle is default of 1 or whatever when only a 4-tuple is specified. This way you don't need the `extra_dashed_lines` and you can also mix and match different line styles within a single collection of `extra_lines`. You could even make them 6-tuples with colors if you see your self extending it further.

Reply:
Good idea. I'll implement your suggestions

Reply:
Added optional extra elements to the tuple to control other aspects

Reply:
nice!